Source/FLHook/HkCbIServerImpl.cpp

```diff
@@ -704,19 +687,12 @@ namespace HkIServerImpl
 
 		EXECUTE_SERVER_CALL(Server.TerminateTrade(iClientID, iAccepted));
 
-		TRY_HOOK {
-			if (iAccepted)
-			{ // save both chars to prevent cheating in case of server crash
```

Comment:
Is it safe to remove this? It seems to me that removing this creates an obvious duplication cheat.

1. Let's say I have characters A and B in game. Character A has 0 credits and character B has 100 million.
1. I use a trade to move 100 million from character B to character A.
1. I then take character A, dock it and log it off, saving those credits.
1. I then park character B in space and wait for some server crash (or do it myself if I know some behaviour that might trigger one), never having the new character file saved.
1. Now I have two characters with 100 million credits.